Sources/xcresultparser/CoberturaCoverageConverter.swift

```diff
@@ -31,6 +31,68 @@
 import Foundation
 import XCResultKit
 
+enum JSONParseError: Error {
+    case convertError(code: Int, message: String)
+}
+
+// Subrange information struct
+struct Subrange: Codable {
```

Comment:
Since all we're doing here is decoding JSON, you could change this (and the ones below) to use `Decodable` instead of `Codable`.
```suggestion
struct Subrange: Decodable {
```